src/application/runner.go

```diff
@@ -1,58 +1,25 @@
 package application
 
 import (
-	"io"
 	"os"
 	"os/signal"
 	"path"
 
-	"github.com/Originate/exosphere/src/config"
-	"github.com/Originate/exosphere/src/docker/composebuilder"
 	"github.com/Originate/exosphere/src/docker/composerunner"
-	"github.com/Originate/exosphere/src/types"
 )
 
-// Runner runs the overall application
-type Runner struct {
-	AppContext               types.AppContext
-	ServiceConfigs           map[string]types.ServiceConfig
-	BuiltDependencies        map[string]config.AppDevelopmentDependency
-	DockerComposeDir         string
-	DockerComposeProjectName string
-	Writer                   io.Writer
-	BuildMode                composebuilder.BuildMode
-}
-
-// NewRunner is Runner's constructor
-func NewRunner(appContext types.AppContext, writer io.Writer, dockerComposeProjectName string, buildMode composebuilder.BuildMode) (*Runner, error) {
-	serviceConfigs, err := config.GetServiceConfigs(appContext.Location, appContext.Config)
-	if err != nil {
-		return &Runner{}, err
-	}
-	allBuiltDependencies := config.GetBuiltDevelopmentDependencies(appContext.Config, serviceConfigs, appContext.Location)
-	return &Runner{
-		AppContext:               appContext,
-		ServiceConfigs:           serviceConfigs,
-		BuiltDependencies:        allBuiltDependencies,
```

Comment:
ServiceConfigs and BuiltDependencies were assgined here and then never used. I would guess these were needed before a refactor that simplified this file.